### PR TITLE
PIM-4942: Display user locale

### PR DESCRIPTION
**Pull request by pierallard**
| Q                 | A
| ----------------- | ---
| Specs             | y
| Behats            | y
| Blue CI           | n
| Changelog updated | n
| Review and 2 GTM  | n
| Micro Demo (PO)   | n
| Migration script  | n
| Tech Doc          | n

### CODE REVIEW

File: features/Context/NavigationContext.php [hunk 1]
```diff
@@ -256,6 +256,14 @@ public function iAmOnMyProfileEditPage()
     }
 
     /**
+     * @Given /^I show my profile$/
```
Comment (solivier) [inline features/Context/NavigationContext.php]: "I am on my profile page" ?

**Comment by grena** [comment]
GTM

**Comment by solivier** [comment]
GTM once tests ok

**Comment by skeleton** [comment]
GTM

**Pull request merged**